src/main/java/org/apache/commons/validator/routines/IBANValidator.java

```diff
@@ -206,13 +215,16 @@ public RegexValidator getRegexValidator() {
             new Validator("SM", 27, "SM\\d{2}[A-Z]{1}\\d{10}[A-Z0-9]{12}"   ), // San Marino
             new Validator("ST", 25, "ST\\d{23}"                             ), // Sao Tome and Principe
             new Validator("SV", 28, "SV\\d{2}[A-Z]{4}\\d{20}"               ), // El Salvador
+            new Validator("TF", 27, "TF\\d{12}[A-Z0-9]{11}\\d{2}"           ), // French Southern Territories
             new Validator("TL", 23, "TL\\d{21}"                             ), // Timor-Leste
             new Validator("TN", 24, "TN\\d{22}"                             ), // Tunisia
             new Validator("TR", 26, "TR\\d{8}[A-Z0-9]{16}"                  ), // Turkey
             new Validator("UA", 29, "UA\\d{8}[A-Z0-9]{19}"                  ), // Ukraine
             new Validator("VA", 22, "VA\\d{20}"                             ), // Vatican City State
             new Validator("VG", 24, "VG\\d{2}[A-Z]{4}\\d{16}"               ), // Virgin Islands
+            new Validator("WF", 27, "WF\\d{12}[A-Z0-9]{11}\\d{2}"           ), // Wallis and Futuna Islands
             new Validator("XK", 20, "XK\\d{18}"                             ), // Kosovo
+            new Validator("YT", 27, "YT\\d{12}[A-Z0-9]{11}\\d{2}"           )  // Mayotte
     };
 
     /** The singleton instance which uses the default formats */
```

Comment:
The information I am finding across the internet is contradictory.  Some places say that the territories in the territories starts with the code of the mother country.  Other places shows examples otherwise.  The French territories were the only ones for which I found something online that specifies the format.  But then again, I have no idea how accurate that site actually is.  Given all that, I can understand why this class was coded based on the IBAN registry, even it if does not appear to match reality.